lib/files.gd

```diff
@@ -316,6 +317,63 @@ DeclareOperation( "ReadTest", [ IsString ] );
 ##
 DeclareOperation( "ReadAsFunction", [ IsString ] );
 
+#############################################################################
+##
+#O  ReadAsExpression( <filename> ) . . . . . . . . read a file as expression
+##
+##  <#GAPDoc Label="ReadAsExpression">
+##  <ManSection>
+##  <Oper Name="ReadAsExpression" Arg='filename'/>
+##
+##  <Description>
+##  reads the file with filename <A>filename</A> as a GAP expression.
+##  <P/>
+##  <E>Example</E>
+##  <P/>
+##  Suppose that the file <F>/tmp/example.g</F> contains the following
+##  <P/>
+##  <Log><![CDATA[
+##  (1,2,3)
+##  ]]></Log>
+##  <P/>
+##  <Log><![CDATA[
+##  gap> a := ReadAsExpression("/tmp/example.g")();
```

Comment:
Where is there an extra () on the end of this line? (Also on end of 'ReadAsList' example)?